data/logs_model/elastic_logs.py

```diff
@@ -51,6 +51,7 @@ class LogEntry(Document):
     repository_id = Integer()
     ip = Ip()
     metadata_json = Text()
```

Comment:
Why keep the existing one around?

Reply:
Figured to just leave it there for now for consistency, since the table model still has a `metadata_json`

Reply:
@josephschorr I removed the field, instead doing a `json.dumps` for the `Log` namedtuple.